src/lando/try_api/api.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from django.core.handlers.wsgi import WSGIRequest
+from django.http import JsonResponse
+from ninja import NinjaAPI
+from ninja.errors import HttpError
+from ninja.security import HttpBearer
+
+from lando.main.auth import LandoOIDCAuthenticationBackend
+
+logger = logging.getLogger(__name__)
+
+
+class GlobalAuth(HttpBearer):
+    def authenticate(self, request: WSGIRequest, token: str) -> User:
```

Comment:
We should have docstrings for the class and `authenticate` methods.

We've talked about moving human users of the automation API to this authentication method in the future. I think this is fine to keep here for now, but we will end up moving it somewhere available to the `try_api` and `headless_api` apps in the future if that happens.